Documentation/Time-Set.md

```diff
@@ -0,0 +1,241 @@
+ * DS3231 register addressed: 0x01
+ */
+
+// Illustrate passing the minutes as a literal value
+myRTC.setMinute(17); // uploads 17 to register 0x01
+```
+
+<h3 id="setHour">void setHour(byte Hour)</h3>
+
+```
+/*
+ * returns: nothing (void)
+ * parameters: byte = 0 to 23 (use 24-hour mode here)
+ * side effects: writes the hour to the DS3231
+ * DS3231 register addressed: 0x02
+ */
+
+// Illustrate passing the hour in a variable
+byte theHour = 19; // equal to 7:00 p.m.
+myRTC.setHour(theHour); // uploads 19 to register 0x02
+```
+
+The setHour() function will convert the hour to 12-hour mode if the DS3231 is set to operate in 12-hour mode. The function does not change how the mode is set on the DS3231 hardware. 
+
+Rather, the mode should be set by another function, setClockMode(), prior to invoking the setHour() function.
+
+<h3 id="setDoW">void setDoW(byte DoW)</h3>
+
+```
+/*
+ * returns: nothing (void)
+ * parameters: byte = 1 to 7
+ * side effects: writes the Day of Week to the DS3231
+ * DS3231 register addressed: 0x03
+ */
+
+// Illustrate passing the Day of the Week as a literal value
+myRTC.setDoW(1); // uploads 1 to register 0x03
+```
+
+The Day of Week value is user-determined. For example, if one chooses Sunday to be the first day of the week, then the DoW values would be:
+
+1. Sunday
+2. Monday
+3. Tuesday. and so forth.
+
+On the other hand, if Monday were selected as the start of a week, then:
+
+1. Monday
+2. Tuesday
+3. Wednesday, etcetera.
+
+<h3 id="setDate">void setDate(byte Date)</h3>
+
+```
+/*
+ * returns: nothing (void)
+ * parameters: byte = 1 to 28, 29, 30, or 31, depending on the month and year
+ * side effects: writes the date of the month to the DS3231
+ * DS3231 register addressed: 0x04
+ */
+
+// Illustrate passing the date in a variable
+byte theDate = 5; // the 5th day of the month
+myRTC.setDate(theDate); // uploads 5 to register 0x04
+```
+
+Reminder: it is the code developer's responsibility to ensure that a sensible value be supplied for the date.
+
+<h3 id="setMonth">void setMonth(byte Month)</h3>
+
+```
+/*
+ * returns: nothing (void)
+ * parameters: byte = 1 to 12
+ * side effects: writes the month to the DS3231
+ * DS3231 register addressed: 0x05
+ */
+
+// Illustrate passing the month in a variable
+byte theMonth = 6; // the 6th month of the year; in English, June
+myRTC.setMonth(theMonth); // uploads 6 to register 0x05
+```
+
+<h3 id="setYear">void setYear(byte Year)</h3>
+
+```
+/*
+ * returns: nothing (void)
+ * parameters: byte = 00 to 99
+ * side effects: writes the year to the DS3231
+ * DS3231 register addressed: 0x06
+ */
+
+// Illustrate passing the year as a literal value
+myRTC.setYear(22); // uploads 22 to register 0x06
+```
+
+<h3 id="setEpoch">void setEpoch(time_t epoch = 0, bool flag_localtime = false)</h3>
+
+Comes now the peculiar concept of the "Epoch" as the word is used by people who write code. It is peculiar because they do not always use it to mean the same thing.
```

Comment:
I find this really strange! I have always used Epoch as the Unix time. I wonder if this has always been this way or if I inadvertently changed it by accepting a PR during a busy time for me without really reading through it.

The DS3231 hardware zeros out on Jan 1st, 2000. So this is a good basis. But I really think of the Epoch as Unix time. Perhaps we can start with this, but then change the library to match this.

I'll check into the code too.